projects/core/bindaas-web-console/src/main/resources/META-INF/templates/administration.vt

```diff
@@ -24,7 +24,7 @@
 		<link rel="stylesheet" href="/foundation/stylesheets/foundation.min.css">
 		<link rel="stylesheet" href="/foundation/stylesheets/app.css">
 		<link rel="stylesheet" href="/foundation/stylesheets/general_enclosed_foundicons.css">
-##		FIXME: Move to proper CSS file later
+##		FIXME: Move to proper CSS file later + ellipsis
```

Comment:
What is the ellipsis in this comment?

Reply:
I changed the CSS for the web-console to show the token as `eYJ...` instead of showing the entire token. This is just a temporary fix and I need to replace it with something better.

Reply:
ok, that's fine.